app/gui/src/main/scala/org/bitcoins/gui/dlc/dialog/AcceptDLCDialog.scala

```diff
@@ -38,7 +43,7 @@ class AcceptDLCDialog
     val fromOffer = OracleInfo.fromTLV(offer.tlv.contractInfo.oracleInfo)
     val fromAnnouncement = SingleOracleInfo(announcement)
```

Comment:
this still seems to be hard-coded to single though no?

Reply:
This is just what we validate against, fixed